### PR TITLE
bowling_challenge

### PR DESCRIPTION
**Pull request by blancopado**

### CODE REVIEW

File: src/game.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	if (pinsDown === 10) {
+			this.rollsArray.push(pinsDown, 0);
+	} else if (this.halfStrike) {
+			//this.currentScore.splice(-1, 1, parseInt(this.currentScore.slice(-1), 10) + pinsDown);
+			this.bonus += pinsDown;
+			this.rollsArray.push(pinsDown);
+	} else {
+			this.rollsArray.push(pinsDown);
+	}
+	console.log(this.rollsArray);
```
Comment (forty9er) [inline src/game.js]: console.log should be removed.

File: src/game.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	console.log(this.spare());
+	console.log(this.currentScore);
+	console.log(this.strikeBonus);
```
Comment (forty9er) [inline src/game.js]: console.log should be removed.

File: src/game.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	this.halfStrike = false;
+	this.bonus = 0;
+	this.strikeBonus = 0;
+};
+
+Game.prototype.rollBall = function(pinsDown) {
+	if (pinsDown === 10) {
+			this.rollsArray.push(pinsDown, 0);
+	} else if (this.halfStrike) {
+			//this.currentScore.splice(-1, 1, parseInt(this.currentScore.slice(-1), 10) + pinsDown);
```
Comment (forty9er) [inline src/game.js]: commented code should be removed

**Comment by tansaku** [comment]
You had your bowling_challenge reviewed by **Andrew Dowell**.
### The good points are:
* Style Guidelines are followed
* Appropriate use of describe blocks, Tests have descriptive titles, Repetition is DRYed to beforeEach functions, Operation of code clear from tests themselves
* Rules are modeled correctly
* Functions are named for the actions they do
* Exceptions used only for exceptional behaviour
* All functions are as short as possible
* Any duck punching is done carefully

### You should consider the following improvements:
* Installation Instructions should be in README
* Unnecessary Components should be Removed
* Little or no testing of edge cases
* Use Spec 'Helpers' Appropriately
* Ensure Asset Routes are Set Correctly
* Poor Encapsulation
* Single Responsibility Principle and other SOLID issues
* Ensure repetition is DRYed out

### Additional comments:
Good attempt!


see https://github.com/makersacademy/bowling_challenge/blob/master/docs/review.md for more details

**Pull request closed without merging**